storage/src/tests/distributor/bucketdbupdatertest.cpp

```diff
@@ -392,7 +392,8 @@ class BucketDBUpdaterTest : public Test,
     }
 
     api::StorageMessageAddress storageAddress(uint16_t node) {
-        return api::StorageMessageAddress("storage", lib::NodeType::STORAGE, node);
+        static vespalib::string _storage;
```

Comment:
Shouldn't this be initialised to _"storage"_ ?

Reply:
Fixed